### PR TITLE
Submissions dssmod 1206

### PR DESCRIPTION
**Pull request by mincho-cb-nytd**
Added submissions for all user

### CODE REVIEW

File: node_modules/cy-verify-downloads/README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
+[![Stand With Ukraine](https://raw.githubusercontent.com/vshymanskyy/StandWithUkraine/main/banner-direct-single.svg)](https://vshymanskyy.github.io/StandWithUkraine)
```
Comment (TylerFSmith2) [inline node_modules/cy-verify-downloads/README.md:1]: Should we keep these kinds of files? Worth a discussion.

File: cypress/e2e/pages/SubmissionsPageObjects.js [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+        fileSearchInput: () => cy.get('[data-testid="textInput"]'),
+        fileSearchInputForSysAdmin: () => cy.get('#transmissionId'),
+        dateRangeName: () => cy.get('[for="date-range-filter-fileReceivedDate"]'),
+        dateRangeInput: () => cy.get('[data-testid="date-range-picker"]'),
+        dateRangeBtn: () => cy.get('[data-testid="daterangepicker_calendar_button"]'),
+        penaltyDropdown: () => cy.get('div.styles_buttonGroup__NYJPa > div:nth-child(1)', { timeout: 10000 }),
+        penaltyDropdownCompliance: () => cy.get('[data-testid="dropdown_wrapper"]').eq(0).find('[data-testid="fieldset"]').children().eq(1).children(),
+        penaltyDropdownNonCompliance: () => cy.get('[data-testid="dropdown_wrapper"]').eq(0).find('[data-testid="fieldset"]').children().eq(2).children(),
+        stateDropdown: () => cy.get(':nth-child(2) > .styles_button__uj25O', { timeout: 10000 }),
+        reportPeriodDropdownForSysadmin: () => cy.get(':nth-child(3) > .styles_button__uj25O', { timeout: 10000 }),
```
Comment (TylerFSmith2) [inline cypress/e2e/pages/SubmissionsPageObjects.js:15]: Change all these "ForSysAdmin" names to fit the files they are used in.

File: cypress/e2e/Submissions/CBCentralSubmissionsPage.js [hunk 3]
```diff
@@ -0,0 +1,233 @@
...
+            expect(option.text()).to.be.oneOf(['Regular', 'Corrected', 'Subsequent', 'Test']);
+        });
+        submissionsPage.elements.statusDropdownForSysAdmin().click();
+        submissionsPage.elements.statusDropdownOptionsForSysAdmin().each((option) => {
+            expect(option.text()).to.be.oneOf(['Hide Inactive Submissions']);
+        });
+        commonPage.elements.clearFiltersBtn().should('contain', 'Clear Filters');
+        commonPage.elements.refreshResultsBtn().should('contain', 'Refresh Results');
+        submissionsPage.elements.fileNumberHeader().should('have.text', 'File Number');
+        cy.get(':nth-child(3) > .styles_tableHeader__mVxy6 > .styles_fieldName__kegXV').should('have.text', 'State');
```
[the remark this answers is not in the document]
Comment (mincho-cb-nytd) [reply cypress/e2e/Submissions/CBCentralSubmissionsPage.js:50]: done

File: cypress/e2e/Submissions/CBCentralSubmissionsPage.js [hunk 4]
```diff
@@ -0,0 +1,233 @@
...
+            return false
+        })
+        cy.visit('/User/Submissions');
+        commonPage.clickOnExportCurrentTable();
+        cy.verifyDownload('nytd_submissions_2024', { contains: true }, { timeout: 70000, interval: 900 });
+    });
+    it("Verify the arrow dropdown button opens the submissions expanded view", function () {
+        cy.visit('/User/Submissions');
+        submissionsPage.elements.firstSubmissionArrowBtn().click();
+        submissionsPage.elements.submissionDetails().should('exist').and('not.have.descendants', '[data-testid="Loading_div"]');
```
[the remark this answers is not in the document]
Comment (mincho-cb-nytd) [reply cypress/e2e/Submissions/CBCentralSubmissionsPage.js:133]: it is verifying the the arrow dropdown is open

File: cypress/e2e/Submissions/CBCentralSubmissionsPage.js [hunk 5]
```diff
@@ -0,0 +1,233 @@
...
+        commonPage.clickOnSubmissionsTabForCBUser();
+        commonPage.verifyBreadCrumbs('Submissions');
+        commonPage.elements.headerH3Text().should('have.text', 'Submissions');
+        submissionsPage.elements.fileSearchName().should('have.text', 'File Search');
+        submissionsPage.elements.submissionDateText().should('have.text', 'Submission Date');
+        submissionsPage.elements.fileSearchInput().should('exist');
+        submissionsPage.elements.dateRangeInput().should('exist');
+        submissionsPage.elements.penaltyDropdown().should('contain', 'Penalty');
+        submissionsPage.elements.stateDropdown().should('contain', 'State');
+        submissionsPage.elements.reportPeriodDropdownForSysadmin().should('contain', 'Report Period');
```
[the remark this answers is not in the document]
Comment (mincho-cb-nytd) [reply cypress/e2e/Submissions/CBCentralSubmissionsPage.js:22]: done replaced to cb central

File: cypress/e2e/Submissions/CBCentralSubmissionsPage.js [hunk 6]
```diff
@@ -0,0 +1,233 @@
...
+        submissionsPage.elements.fileTypeDropdownForSysAdmin().should('contain', 'File Type');
```
[the remark this answers is not in the document]
Comment (mincho-cb-nytd) [reply cypress/e2e/Submissions/CBCentralSubmissionsPage.js:23]: replaced to cb central locator

File: cypress/e2e/Submissions/CBCentralSubmissionsPage.js [hunk 7]
```diff
@@ -0,0 +1,233 @@
...
+        submissionsPage.elements.statusDropdownForSysAdmin().should('contain', 'Status');
```
[the remark this answers is not in the document]
Comment (mincho-cb-nytd) [reply cypress/e2e/Submissions/CBCentralSubmissionsPage.js:24]: done

File: cypress/e2e/Submissions/CBCentralSubmissionsPage.js [hunk 8]
```diff
@@ -0,0 +1,233 @@
...
+        submissionsPage.elements.submissionDetails().children().eq(2).find('tbody').should('contain', 'Served').should('contain', 'Baseline').should('contain', 'Follow-up').should('contain', 'Total');
+        submissionsPage.elements.submissionDetails().children().eq(3).children().eq(0).should('have.text', 'Meets Standards');
+        submissionsPage.elements.submissionDetails().children().eq(3).find('tbody').should('contain', 'File Submission').should('contain', 'Data');
+        submissionsPage.elements.submissionDetails().children().eq(4).children().eq(0).should('have.text', 'Data Quality Advisories');
+        submissionsPage.elements.submissionDetails().children().eq(4).find('tbody').should('contain', 'Element-level').should('contain', 'Record-level').should('contain', 'Total');
+        submissionsPage.elements.submissionDetails().children().eq(5).children().eq(0).should('have.text', 'Element Compliance');
+        submissionsPage.elements.submissionDetails().children().eq(4).find('tbody').should('contain', 'Total');
+        submissionsPage.elements.submissionDetails().children().eq(5).children().eq(0).should('have.text', 'Element Compliance');
```
[the remark this answers is not in the document]
Comment (mincho-cb-nytd) [reply cypress/e2e/Submissions/CBCentralSubmissionsPage.js:143]: removed dublication

File: cypress/e2e/Submissions/DefaultStateSubmissionsPage.js [hunk 9]
```diff
@@ -0,0 +1,225 @@
...
+        submissionsPage.elements.reportPeriodDropdown().should('contain', 'Report Period');
+        submissionsPage.elements.fileTypeDropdown().should('contain', 'File Type');
+        submissionsPage.elements.statusDropdown().should('contain', 'Status');
+        submissionsPage.elements.penaltyDropdown().click();
+        submissionsPage.elements.penaltyDropdownCompliance().should('have.text', 'Compliant0.00%');
+        submissionsPage.elements.penaltyDropdownNonCompliance().each((option) => {
+            expect(option.text()).to.be.oneOf(['Non-Compliant', '0.00%', 'All', '0.50%', '1.00%', '1.25%', '1.50%', '1.75%', '2.50%']);
+        });
+        submissionsPage.elements.reportPeriodDropdown().click();
+        submissionsPage.elements.reportPeriodDropdownOptionsForSysAdmin().each((option) => {
```
[the remark this answers is not in the document]
Comment (mincho-cb-nytd) [reply cypress/e2e/Submissions/DefaultStateSubmissionsPage.js:30]: Added for cbCentral

File: cypress/fixtures/uploadExample.xml [hunk 10]
```diff
@@ -0,0 +1,17 @@
+<root>
```
[the remark this answers is not in the document]
Comment (mincho-cb-nytd) [reply cypress/fixtures/uploadExample.xml]: ignore this file I was working on upload functionality I suppose to delete it

File: cypress/e2e/pages/SubmissionsPageObjects.js [hunk 11]
```diff
@@ -0,0 +1,143 @@
+import { timeInterval } from "rxjs";
+
+class SubmissionsPageObjects {
+    elements = {
+        fileSearchName: () => cy.get('[for="transmissionId"]'), // Name Search label
+        fileSearchInput: () => cy.get('[data-testid="textInput"]'),
+        fileSearchInputForSysAdmin: () => cy.get('#transmissionId'),
```
[the remark this answers is not in the document]
Comment (mincho-cb-nytd) [reply cypress/e2e/pages/SubmissionsPageObjects.js:7]: For System admin and other user the element is deferent

File: node_modules/cy-verify-downloads/README.md (hunk 1 above)
Comment (mincho-cb-nytd) [reply node_modules/cy-verify-downloads/README.md]: This read me comes with cy-verify-downlaods

File: cypress/e2e/pages/SubmissionsPageObjects.js (hunk 2 above)
Comment (mincho-cb-nytd) [reply cypress/e2e/pages/SubmissionsPageObjects.js:15]: done

**Pull request merged**